sinks/kafka/driver_test.go

```diff
@@ -0,0 +1,237 @@
+		},
+		{
+			Reason:        event2Reason,
+			LastTimestamp: event2Time,
+			Source: kube_api.EventSource{
+				Host: event2SourceHostname,
+			},
+		},
+	}
+	err := fakeSink.StoreEvents(events)
+	assert.NoError(t, err)
+	assert.Equal(t, 2, len(fakeSink.fakeProducer.msgs))
+	events0Json, _ := json.Marshal(events[0])
+	events1Json, _ := json.Marshal(events[1])
+	assert.Equal(t, string(events0Json), fakeSink.fakeProducer.msgs[0].message)
+	assert.Equal(t, string(events1Json), fakeSink.fakeProducer.msgs[1].message)
+}
+
+func TestStoreTimeseriesNilInput(t *testing.T) {
+	fakeSink := NewFakeSink()
+	err := fakeSink.StoreTimeseries(nil)
+	assert.NoError(t, err)
+	assert.Equal(t, 0, len(fakeSink.fakeProducer.msgs))
+}
+
+func TestStoreTimeseriesEmptyInput(t *testing.T) {
+	fakeSink := NewFakeSink()
+	err := fakeSink.StoreTimeseries([]sink_api.Timeseries{})
+	assert.NoError(t, err)
+	assert.Equal(t, 0, len(fakeSink.fakeProducer.msgs))
+}
+
+func TestStoreTimeseriesSingleTimeserieInput(t *testing.T) {
+	fakeSink := NewFakeSink()
+
+	smd := sink_api.MetricDescriptor{
+		ValueType: sink_api.ValueInt64,
+		Type:      sink_api.MetricCumulative,
+	}
+
+	l := make(map[string]string)
+	l["test"] = "notvisible"
+	l[sink_api.LabelHostname.Key] = "localhost"
+	l[sink_api.LabelContainerName.Key] = "docker"
+	l[sink_api.LabelPodId.Key] = "aaaa-bbbb-cccc-dddd"
+
+	p := sink_api.Point{
+		Name:   "test/metric/1",
+		Labels: l,
+		Start:  time.Now(),
+		End:    time.Now(),
+		Value:  int64(123456),
+	}
+
+	timeseries := []sink_api.Timeseries{
+		{
+			MetricDescriptor: &smd,
+			Point:            &p,
+		},
+	}
+
+	err := fakeSink.StoreTimeseries(timeseries)
+	assert.NoError(t, err)
+	assert.Equal(t, 1, len(fakeSink.fakeProducer.msgs))
+	timeseries1Json, _ := json.Marshal(timeseries[0])
+	assert.Equal(t, string(timeseries1Json), fakeSink.fakeProducer.msgs[0].message)
+}
+
+func TestStoreTimeseriesMultipleTimeseriesInput(t *testing.T) {
+	fakeSink := NewFakeSink()
+
+	smd := sink_api.MetricDescriptor{
+		ValueType: sink_api.ValueInt64,
+		Type:      sink_api.MetricCumulative,
+	}
+
+	l := make(map[string]string)
+	l["test"] = "notvisible"
+	l[sink_api.LabelHostname.Key] = "localhost"
+	l[sink_api.LabelContainerName.Key] = "docker"
+	l[sink_api.LabelPodId.Key] = "aaaa-bbbb-cccc-dddd"
+
+	p1 := sink_api.Point{
+		Name:   "test/metric/1",
+		Labels: l,
+		Start:  time.Now(),
+		End:    time.Now(),
+		Value:  int64(123456),
+	}
+
+	p2 := sink_api.Point{
+		Name:   "test/metric/1",
+		Labels: l,
+		Start:  time.Now(),
+		End:    time.Now(),
+		Value:  int64(123456),
+	}
+
+	timeseries := []sink_api.Timeseries{
+		{
+			MetricDescriptor: &smd,
+			Point:            &p1,
+		},
+		{
+			MetricDescriptor: &smd,
+			Point:            &p2,
+		},
+	}
+
+	err := fakeSink.StoreTimeseries(timeseries)
+	assert.NoError(t, err)
+	assert.Equal(t, 2, len(fakeSink.fakeProducer.msgs))
+	timeseries0Json, _ := json.Marshal(timeseries[0])
+	timeseries1Json, _ := json.Marshal(timeseries[1])
```

Comment:
I'm not sure whether ignoring errors here makes sense. Maybe panic if `err != nil`, similar to `regexp.MustCompile`?